WORKSPACE

```diff
@@ -16,11 +16,17 @@ go_repositories()
 
 go_repository(
     name = "com_google_cloud_go",
-    commit = "57377bad3486b37af17b47230a61603794c798ae",
+    commit = "3b1ae45394a234c385be014e9a488f2bb6eef821",
```

Comment:
Do you want this specific SHA, how old is this commit? Maybe try something newer (https://code.googlesource.com/gocloud/) and toolbox/pkg_check will work just fine.